js/app.js

```diff
@@ -353,21 +350,7 @@ const main = () => {
           $scope.songs = data.tracks;
           $scope.current_list_id = list_id;
 
-          $timeout(() => {
-            // use timeout to avoid stil in digest error.
-            angularPlayer.clearPlaylist((res) => {
-              // add songs to playlist
-              angularPlayer.addTrackArray($scope.songs);
-              // play first song
-              let index = 0;
-              if (angularPlayer.getShuffle()) {
-                const max = $scope.songs.length - 1;
-                const min = 0;
-                index = Math.floor(Math.random() * (max - min + 1)) + min;
-              }
-              angularPlayer.playTrack($scope.songs[index].id);
-            });
-          }, 0);
+          l1Player.setNewPlaylist($scope.songs);
```

Comment:
add `l1Player.play()` so when click play button in home page playlist cover, the music will play directly.